obspy/io/cybershake/core.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""
+Support for reading CyberShake seismograms in ObsPy.
+"""
+
+import struct
+import numpy as np
+from obspy import Trace, Stream
+
+
+NET_CODE = 'CS'  # Assign a network code for CyberShake data
+BAND_INST_CODE = 'MX'  # M indicates mid-period and X indicates synthetic data
+LOCATION_CODE = '00'
```

Comment:
Are these values good for all data that might be in this file format? Another option would be to add these as kwargs to `_read_cybershake()`, then users could pass custom values through via a `read()` call.
(Even though people could of course always change that later on modifying the traces Stats attributes)

Also there is no real reason to save space in the length of these variable names here, so it would be good to have fully explicit names `NETWORK_CODE` and `BAND_INSTRUMENT_CODE` to not leave any room for ambiguity. Explicit is better than implicit, see Python Zen :)

Reply:
It is possible one could have data using a different period band, so I changed it to allow for a user-specified `BAND_CODE` but the default is still 'M'. I also spelled out the full variable names now.